src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponent.scala

```diff
@@ -49,33 +50,34 @@ trait ClusterComponent extends LeoComponent {
-    def jupyterUserScriptUri =        column[Option[String]]    ("jupyterUserScriptUri",  O.Length(1024))
-    def initBucket =                  column[Option[String]]    ("initBucket",            O.Length(1024))
-    def serviceAccountKeyId =         column[Option[String]]    ("serviceAccountKeyId",   O.Length(254))
-    def stagingBucket =               column[Option[String]]    ("stagingBucket",         O.Length(254))
-    def dateAccessed =                column[Timestamp]         ("dateAccessed",          O.SqlType("TIMESTAMP(6)"))
-    def autopauseThreshold =                      column[Int]                    ("autopauseThreshold")
-    def defaultClientId =                               column[Option[String]]      ("defaultClientId",     O.Length(1024))
+    def id = column[Long]("id", O.PrimaryKey, O.AutoInc)
+    def clusterName = column[String]("clusterName", O.Length(254))
+    def googleId = column[Option[UUID]]("googleId")
+    def googleProject = column[String]("googleProject", O.Length(254))
+    def clusterServiceAccount = column[Option[String]]("clusterServiceAccount", O.Length(254))
+    def notebookServiceAccount = column[Option[String]]("notebookServiceAccount", O.Length(254))
+    def numberOfWorkers = column[Int]("numberOfWorkers")
+    def masterMachineType = column[String]("masterMachineType", O.Length(254))
+    def masterDiskSize = column[Int]("masterDiskSize")
+    def workerMachineType = column[Option[String]]("workerMachineType", O.Length(254))
+    def workerDiskSize = column[Option[Int]]("workerDiskSize")
+    def numberOfWorkerLocalSSDs = column[Option[Int]]("numberOfWorkerLocalSSDs")
+    def numberOfPreemptibleWorkers = column[Option[Int]]("numberOfPreemptibleWorkers")
+    def operationName = column[Option[String]]("operationName", O.Length(254))
+    def status = column[String]("status", O.Length(254))
+    def hostIp = column[Option[String]]("hostIp", O.Length(254))
+    def creator = column[String]("creator", O.Length(254))
+    def createdDate = column[Timestamp]("createdDate", O.SqlType("TIMESTAMP(6)"))
+    def destroyedDate =  column[Timestamp]("destroyedDate", O.SqlType("TIMESTAMP(6)"))
+    def jupyterExtensionUri = column[Option[String]]("jupyterExtensionUri", O.Length(1024))
+    def jupyterUserScriptUri = column[Option[String]]("jupyterUserScriptUri", O.Length(1024))
+    def initBucket = column[Option[String]]("initBucket", O.Length(1024))
+    def serviceAccountKeyId= column[Option[String]]("serviceAccountKeyId", O.Length(254))
+    def stagingBucket = column[Option[String]]("stagingBucket", O.Length(254))
+    def dateAccessed= column[Timestamp]("dateAccessed", O.SqlType("TIMESTAMP(6)"))
+    def autopauseThreshold = column[Int]("autopauseThreshold")
+    def defaultClientId = column[Option[String]]("defaultClientId", O.Length(1024))
+    def stopAfterCreation = column[Boolean]("stopAfterCreation")
```

Comment:
Line#80 is the only change above. Previously, the line segments were horribly rendered on my IntelliJ so I wanted to at least standardize them. GitHub turned out to display them nicely somehow.